mkdocs.yml

```diff
@@ -297,6 +296,4 @@ plugins:
     # https://github.com/mkdocs/mkdocs-redirects
     - redirects:
         redirect_maps:
-        #  'old.md': 'new.md'
-        #  'old/file.md': 'new/file.md'
-        #  'some_file.md': 'http://external.url.com/foobar'
+            'concepts/manifest.md': 'concepts/environments.md#manifesttoml'
```

Comment:
suggestion nonblocking: should this redirect to the man page? It's a bit weird to click a link that tells you to click another link

followup:
Will this link in the CLI still work?
https://flox.dev/docs/concepts/manifest/#package-descriptors

Reply:
> I'm not sure what you mean by "click a link that tells you to click another link", the browser automatically redirects you from `concepts/manifest` to `concepts/environments/#manifesttoml`.

`concepts/environments/#manifesttoml` says
```
See [`manifest.toml`][manifest] for a complete description of the manifest format and the [customizing environments guide][customizing_environments_guide] to walk through examples.
```